specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-preview/examples/ReservationOrderByBillingAccount.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "api-version": "2020-11-01-preview",
+    "billingAccountName": "{billingAccountName}",
+    "reservationOrderId": "{reservationOrderId}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/reservationOrders/{reservationOrderId}",
+        "type": "microsoft.billing/reservationOrders",
+        "name": "Reservation Name",
+        "properties": {
+          "productType": "AzureHybridBenefit",
+          "tenantId": "00000000-0000-0000-0000-000000000000",
+          "effectiveDate": "2020-07-30T18:36:50.7939106Z",
+          "requestPurchaseTime": "0001-01-01T00:00:00",
+          "expiryDate": null,
+          "reviewDate": null,
+          "endDate": null,
+          "term": "None",
+          "reservationName": "SQL_Server_Enterprise_Edition",
+          "billingScope": "",
+          "billingPlanDetails": null,
+          "state": "Created",
+          "subState": "Active",
+          "reservations": [
```

Comment:
do you really need to surface reservations in the response of reservationOrders? The customer can do an explicit GET to get all reservations under a reservationOrder.

Reply:
We did the same thing in Microsoft.Capacity for the reservationOrder. https://github.com/Azure/azure-rest-api-specs/blob/e0b78897850ccbb648b3efe286b78e78c5cd8bcf/specification/reservations/resource-manager/Microsoft.Capacity/stable/2017-11-01/reservations.json#L737